confignode/src/main/java/org/apache/iotdb/confignode/partition/DataPartitionInfo.java

```diff
@@ -18,63 +18,26 @@
  */
 package org.apache.iotdb.confignode.partition;
 
-import java.util.ArrayList;
-import java.util.HashMap;
+import org.apache.iotdb.commons.partition.DataRegionReplicaSet;
+import org.apache.iotdb.commons.partition.TimePartitionId;
+
 import java.util.List;
 import java.util.Map;
 
 public class DataPartitionInfo {
```

Comment:
This class should be deleted and use org.apache.iotdb.commons.partition.DataPartitionInfo instead.